src/routes/(app)/elections/ElectionEditor.svelte

```diff
@@ -4,18 +4,16 @@
   import { superForm } from "$lib/utils/client/superForms";
   import type { ElectionSchema } from "./schemas";
   import * as m from "$paraglide/messages";
+  import type { Election } from "@prisma/client";
 
   export let isCreating: boolean;
   export let data: {
     form: SuperValidated<ElectionSchema>;
     committees: Array<{ id: string; name: string; nameEn: string | null }>;
-    election: {
-      markdown: string;
-      markdownEn: string | null;
-      link: string;
-      expiresAt: Date;
-      committeeId: string;
-    };
+    election: Pick<
+      Election,
+      "markdown" | "markdownEn" | "link" | "expiresAt" | "committeeId"
+    >;
```

Comment:
I changed to to use TypeScript's `Pick` utility type so that we can reuse `Election` type that already exists.